UI/window-basic-filters.cpp

```diff
@@ -275,9 +275,11 @@ void OBSBasicFilters::UpdatePropertiesView(int row, bool async)
 	if (view) {
 		updatePropertiesSignal.Disconnect();
 		ui->propertiesFrame->setVisible(false);
-		view->hide();
-		view->deleteLater();
-		view = nullptr;
+		if (view) {
```

Comment:
I think this could use an explanatory comment before it gets removed in a few years because "view was already null checked".

Reply:
Funnily enough the commit message started its life as a code comment and I thought to myself "well I explained it in the commit message", but doesn't hurt to add it here as well.